### ISSUE TITLE
Integration tests are missing important focus management tests

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly-react#6343]
We need working tests to evaluate focus management in two scenarios:

1. keyboard handling of focus in menu and menu like components
2. aria-disabled/disabled items - aria-disabled should be able to get browser focus but not handle click events

Previous attempts to test these features have been removed when cypress upgraded. Cypress does not like us clicking on disabled items or executing events on items that don't already have focus.

**Comment by thatblindgeye** [comment]
Due to inactivity I'm going to close this out for now, but moved to Parking Lot to keep in mind that it's something to revisit when we have the bandwidth.
